pkg/integrations/v2/app_o11y_receiver/exporters/logs_test.go

```diff
@@ -0,0 +1,120 @@
+package exporters
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"strings"
+	"testing"
+	"time"
+
+	kitlog "github.com/go-kit/log"
+	"github.com/grafana/agent/pkg/integrations/v2/app_o11y_receiver/models"
+	"github.com/grafana/loki/clients/pkg/promtail/api"
+	prommodel "github.com/prometheus/common/model"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
nit: we tend to use /require instead of assert, since most assertions that are being made are blockers for the rest of the test to pass.

Reply:
- [ ] use `/require` instead